packages/react-ai/src/components/AIConversation/context/ResponseComponentsContext.tsx

```diff
@@ -4,9 +4,22 @@ import { ToolConfiguration } from '../../../types';
 
 type ResponseComponentsContextProps = ResponseComponents | undefined;
 
+export const RESPONSE_COMPONENT_PREFIX = 'UI_';
+
 export const ResponseComponentsContext =
   React.createContext<ResponseComponentsContextProps>(undefined);
 
+const prependResponseComponents = (responseComponents?: ResponseComponents) => {
```

Comment:
This makes it so that we have to run this utility every time responseComponents changes, which responseComponents is set on the `createAIConversation` so it shouldnt be changing much if at all